automation/test.sh

```diff
@@ -17,6 +17,29 @@
 # Copyright 2018 Red Hat, Inc.
 #
 
+# Start CPU manager only for templates which require it.
```

Comment:
Why is the CPU manager started only for certain templates? Is it only a performance optimization, so the tests take less time? If yes, how much slower are tests with CPU manager?

I don't like that there is a list of template names in this test script. If the performance hit is not too big, the CPU manager could be enabled for all templates.

Reply:
The cpu manager is allowed only for templates, which have high performance variant. Other templates don't need it, because they don't have HP variant. I did it this way, because it takes about 10 minutes to enable cpu manager on nodes.

Reply:
Ok, that makes sense.
Can you write this information to the comment? So it is clear why is it needed only for these templates. And maybe can you also mention the specific feature of HP VMs that requires the CPU manager?

Reply:
done